deb/Makefile

```diff
@@ -1,7 +1,7 @@
 all: deb-build
 
 clean:
-	rm -rf output brave-keyring/etc/apt/trusted.gpg.d/* *.asc
```

Comment:
After building the package on my machine and looking inside the `.deb` manually, I found that it still contained
`/etc/apt/trusted.gpg.d/brave-browser-release.gpg`. That's due to the fact that I had that file lying around locallly and that `make clean` didn't delete it.

I'd suggest keeping `brave-keyring/etc/apt/trusted.gpg.d` in the `clean` target to make sure that nobody accidentally ships it by using an old build directory or git branch.

Reply:
That's a good point. I changed the recipe.

Keep in mind though, because docker is used for building and the user inside the container is root, files end up being created with root ownership and calling `make clean` may not remove them anyway (`sudo make clean` would, but that's far from ideal).

The proper thing to do would be to run as an unprivileged user in the container, but I think that's out of scope for this issue.